02week/bio/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Nick Andrew Gonzalez Biography</title>
+  <link href="css/style.css" rel="stylesheet" type="text/css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+  <h1>Meet Nick</h1>
+  <div class="icon-bar">
+  <a href="#"><i class="fa fa-facebook-square" aria-hidden="true"></i>
+  <a href="#"><i class="fa fa-instagram" aria-hidden="true"></i>
+  <a href="#"><i class="fa fa-youtube-play" aria-hidden="true"></i>
+  <a href="#"><i class="fa fa-envelope"></i></a>
+  </div>
+  <img src="images/Resumepic.jpg" width="450px" height="450px" alt="Resumepic">
```

Comment:
Now that you know CSS, this in-line setting of height/width are not allow. make sure that you use CSS for this